src/all/mangadex/src/eu/kanade/tachiyomi/extension/all/mangadex/MangaDex.kt

```diff
@@ -757,11 +758,27 @@ abstract class MangaDex(final override val lang: String, private val dexLang: St
             }
         }
 
+        val preferExtensionLangTitlePref = SwitchPreferenceCompat(screen.context).apply {
+            key = MDConstants.getPreferExtensionLangTitlePrefKey(dexLang)
+            title = helper.intl["prefer_title_in_extension_language"]
+            summary = helper.intl["prefer_title_in_extension_language_summary"]
+            setDefaultValue(true)
```

Comment:
It makes sense to enable for new users, but not at the cost of invalidating downloads for existing users. Do you think detecting this would be doable? Otherwise the default should be false.